pkg/serviceimport/map.go

```diff
@@ -148,7 +148,7 @@ func (m *Map) Put(serviceImport *mcsv1a1.ServiceImport) {
 		}
 
 		if serviceImport.Spec.Type == mcsv1a1.ClusterSetIP {
-			clusterName := serviceImport.GetLabels()[lhconstants.LabelSourceCluster]
+			clusterName := serviceImport.GetLabels()[lhconstants.LighthouseLabelSourceCluster]
```

Comment:
Shouldn't this use `MCSLabelSourceCluster`?

Reply:
Not on a ServiceImport; the new labels are for EndpointSlices only.